src/stream.ts

```diff
@@ -300,7 +300,7 @@ export class DataAndMoneyStream extends Duplex {
   async receiveTotal (limit: BigNumber.Value, opts?: ReceiveOpts): Promise<void> {
     const timeout = (opts && opts.timeout) || DEFAULT_TIMEOUT
     if (this._totalReceived.isGreaterThanOrEqualTo(limit)) {
-      this.debug(`already received ${this._totalReceived}, not waiting for more`)
+      this.log.trace(`already received ${this._totalReceived}, not waiting for more`)
```

Comment:
I think debug would be good here since its output in limited case but reflects the total received when output.